.github/workflows/pyotracker.yml

```diff
@@ -17,19 +17,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        pyomo-version: [6.4.2, main]
+        pyomo-version: [6.6.1, main]
```

Comment:
Are we no longer compatible with Pyomo 6.4.2?

The original idea here was to try to cover the oldest release we are compatible with to the bleeding edge.

Reply:
OK. I'll switch it back to 6.4.2

Reply:
Ben,  Can you update the doc test to Python 3.9?